### PR TITLE
Feat/5655  remove raw_content from events

### PR DESCRIPTION
**Pull request by inkhey**
related to #5655 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

Please, test:
- Edit/Delete/Restore a note
- TLMs when another user edit/delete/restore a note
- Create/Edit/Delete a comment
- TLMs when another user create/edit/delete a comment
- Create/Edit/Delete/Restore a news
- TLMs when another user create/edit/delete/restore a news
- See in your console if `content.*.*` TLMs do not have the `raw_content` field

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -13,7 +13,7 @@ develop-eggs/
 eggs/
 #lib/
 lib64/
-parts/
+parttest_api__update_folder__ok_200__nominal_cases/
```
Comment (inkhey) [inline backend/.gitignore:16]:
```suggestion
parts/
```

File: backend/tracim_backend/views/core_api/schemas.py [hunk 2]
```diff
@@ -1808,7 +1811,7 @@ class PreviewInfoSchema(marshmallow.Schema):
     )
 
 
-class FileContentSchema(ContentSchema):
+class MessageFileContentSchem(ContentSchema):
```
Comment (Haskaris) [inline backend/tracim_backend/views/core_api/schemas.py:1814]: Is it a typo/mistake that is `Schem` and not `Schema`?

**Review by Haskaris — APPROVED** [review]

File: backend/tracim_backend/views/core_api/schemas.py (hunk 2 above)
Comment (inkhey) [reply backend/tracim_backend/views/core_api/schemas.py:1814]: yes ...

Comment (inkhey) [reply backend/tracim_backend/views/core_api/schemas.py]: fixed

**Comment by Haskaris** [comment]
Tested on branch.

**Pull request merged**